dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImplTest.java

```diff
@@ -57,15 +64,39 @@ public static void prepare() throws Exception{
 
         site = new SiteDataGen().nextPersisted();
     }
-
-    /*@AfterClass
-    public static void cleanup() throws DotDataException, DotSecurityException {
-
-        cleanupDebug(ESContentFactoryImplTest.class);
-    }*/
     
     final ESContentFactoryImpl instance = new ESContentFactoryImpl();
 
+    public static class TestCase {
+
+        String formatPattern;
+
+        public TestCase(final String formatPattern) {
+            this.formatPattern = formatPattern;
+        }
+    }
+
+    @DataProvider
+    public static Object[] testCases() {
+        return new TestCase[]{
+                new TestCase("MM/dd/yyyy hh:mm:ssa"),
+                new TestCase("MM/dd/yyyy hh:mm:ss a"),
+                new TestCase("MM/dd/yyyy hh:mm a"),
+                new TestCase("MM/dd/yyyy hh:mma"),
+                new TestCase("MM/dd/yyyy HH:mm:ss"),
+                new TestCase("MM/dd/yyyy HH:mm"),
+                new TestCase("yyyyMMddHHmmss"),
+                new TestCase("yyyyMMdd"),
+                new TestCase("MM/dd/yyyy"),
+                new TestCase("hh:mm:ssa"),
+                new TestCase("hh:mm:ss a"),
+                new TestCase("HH:mm:ss"),
+                new TestCase("hh:mma"),
+                new TestCase("hh:mm a"),
+                new TestCase("HH:mm")
```

Comment:
These different formats can be placed into a list somewhere, visible for testing, so both the class and the test get them from the same place and avoid mismatching if it gets modified.

Reply:
I think is not good idea because if by mistake one is deleted the the test keep passing

Reply:
Done